src/engraving/dom/excerpt.cpp

```diff
@@ -344,19 +344,19 @@ void Excerpt::createExcerpt(Excerpt* excerpt)
 
     cloneStaves(masterScore, score, srcStaves, excerpt->tracksMapping());
 
+    MeasureBase* masterMeasure = masterScore->first();
     MeasureBase* scoreMeasure = score->first();
 
-    if (!scoreMeasure || !scoreMeasure->isVBox()) {
+    if ((!scoreMeasure || !scoreMeasure->isVBox()) && !masterMeasure->excludeFromOtherParts()) {
```

Comment:
Note: this causes a crash when creating parts for a score that contains no measures nor boxes. Although 4.1.1 also shows some funny behaviour in that situation, it does not crash, so this will be a regression.

(just making a note of it so that we can log/fix it later)